spec/controller/posts_spec.rb

```diff
@@ -0,0 +1,52 @@
+# including the library
+require 'rails_helper'
+
+# Test 1
+# It is for Post Controller
+RSpec.describe PostsController, type: :controller do
+  controller do
+    # checking index method
+    def index
+      # it should be status OK
+      render json: {}, status: 200
+    end
+  end
+
+  # Test 2
+  describe 'Check for customn status' do
+    it 'Returns a 1 custom status code' do
+      get :index
+      expect(response).to have_http_status(1)
+    end
+  end
```

Comment:
- [ ] This test is failing because you defined status `200` but in the test, you are testing status for `1`. Please fix this issue and make sure your status code and test have the same values.

Reply:
done.
![image](https://user-images.githubusercontent.com/13853450/192115793-abaef596-b6ad-408f-84f2-6bd3b51e0ca6.png)